shared/oae/bundles/ui/default.properties

```diff
@@ -509,4 +511,4 @@ WORD_DOCUMENT = Word document
 XML_DOCUMENT = XML document
 YOUR_PASSWORD_SHOULD_BE_AT_LEAST_SIX_CHARACTERS_LONG = Your password should be at least 6 characters long
 YOU_MAY_NOT_HAVE_ACCESS_TO_VIEW_THE_PAGE_LOGGED_OUT = You might have access to this page after signing in
-
+EMAIL_RESET_SUBJECT = Reset Your Password
```

Comment:
We might want to clarify which service the password reset is for in the email subject.